### DISCUSSION TITLE
Percent of Top pages based on current filter

### DISCUSSION THREAD

**Discussion by SemaphoreOxalis** [plausible/analytics#1764] [Anything Else]
Hi,
My users love Plausible BUT (there always is a but ^^) they report the loss of the percent of top page view based on the current filter with is usefull for "basic" stats. I look at the docs without finding it so I decide to post here my asking ;-)

Sorry if there is a way to get it.

**Comment by metmarkosaric** [comment]
thanks for the feedback @SemaphoreOxalis! for sure! would be nice to add the percentage to the top graph on filtered views. there's a thread for this idea already: https://github.com/plausible/analytics/discussions/365

**Comment by SemaphoreOxalis** [comment]
Ho sorry, I have make a search on "Percent" and weirdly found no topic about that. So I will wait for the release of this ;-)
